app/controllers/api/products_controller.rb

```diff
@@ -136,6 +142,10 @@ def render_paged_products(products)
       }.to_json
     end
 
+    def query_params_with_defaults
+      params[:q].to_h.reverse_merge!('s' => 'created_at desc')
```

Comment:
I'd rather not mutate to avoid nasty side effects with the params so I'd rather use `#reverse_merge` instead of `#reverse_merge!`

Reply:
Done